scripts/wof-find-property

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# -*-python-*-
+
+import sys
+import os.path
+import logging
+import geojson
+import re
+import mapzen.whosonfirst.geojson
+
+def find_property(path, options):
+    try:
+        fh = open(path, 'r')
+        f = geojson.load(fh)
+        fh.close()
+
+        if options.prefix:
+            regex = '^{}:'.format(options.prefix)
+            pattern = re.compile(regex)
+            for key, value in f['properties'].iteritems():
+                if re.search(pattern, key):
+                    p, filename = os.path.split(path)
+                    if isinstance(value, str) or isinstance(value, unicode):
+                        value = value.encode('utf-8')
+
+                    print('{} {} - {} ({})'.format(filename, key, value, type(value)))
+
+        elif options.property in f['properties']:
+            p, filename = os.path.split(path)
+            value = f['properties'][options.property]
+            if isinstance(value, str) or isinstance(value, unicode):
+                value = value.encode('utf-8')
+
+            print('{} {} - {} ({})'.format(filename, options.property, value, type(value)))
+
+    except:
+        print('\nFatal exception caught in {}'.format(path))
```

Comment:
For warnings and errors, we should use the `logging` facilities.

Reply:
Agreed; fixed in commit `6590205`